README.md

```diff
@@ -19,26 +19,39 @@
 
 It is meant to bridge the gap between the different uses of your algorithms. With `modelkit` you can ensure that the same exact code will run in production, on your machine, or on data processing pipelines.
 
-## Features
+## Quickstart
 
-`modelkit`'s key features are:
+`modelkit` provides a straightforward and consistent way to wrap your prediction  code in a `Model` class:
 
-- **simple** `modelkit` is just a Python library, use pip to install it and you are done.
-- **custom** `modelkit` is useful whenever you need to go beyond off-the-shelf models: custom processing, heuristics, business logic, different frameworks, etc.
-- **framework agnostic** you bring your own framework to the table, and you can use whatever code or library you want. Similarly, `modelkit` is not opinionated about how you build or train your models.
-- **organized** `modelkit` encourages you to version and share you ML library and artifacts with others, as a Python package or as a service. Let others use and evaluate your models!
-- **fast** `modelkit` add minimal overhead to prediction calls. Model predictions can be batched for speed (you define the batching logic).
-- **fast to code** Models only need to define their prediction logic and that's it. No cumbersome pre or postprocessing logic, branching options, etc... The boilerplate code is minimal and sensible.
-- **fast to deploy** Models can be served in a single CLI call using [fastapi](https://fastapi.tiangolo.com/)
+```python
+from modelkit import Model
+
+class MyModel(Model):
```

Comment:
This quickstart example might be too simplistic to understand what modelkit really is about, and make anyone wants to use it. 

We could illustratre even more the features described below using the tutorial we already wrote ?

Reply:
I am trying to keep it as short as possible, but I do realise that it may well be overly simple. I wonder whether we should say something like:
"Now you can:
- expose it as an API with a single CLI
- use it in your code
- use it in batch
- share it"
Let's discuss this offline